packages/systemd/var-run-tmpfiles.conf

```diff
@@ -0,0 +1,4 @@
+d /run/cache 0755 root root -
```

Comment:
Is `/run/cache` symlinked at `/var/cache` by another process or config? Or should this set of tmpfiles rules include an entry for `/var/cache`?

Reply:
`/run/cache` is used by our updates API, and also for the ldconfig cache.

`/var/cache` is separate. It gets created as a side effect of the tmpfiles snippet for `/var/cache/private`, but would also be created by updog if it wasn't present.